### PR TITLE
Fix minor style

### PR DESCRIPTION
**Pull request by hjmjohnson**
Spelling and minor changes.

## PR Checklist
- [X] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) was made (or the changes have been approved)
- [X] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)
- [X] Added test (or behavior not changed)

### CODE REVIEW

File: Modules/Core/Common/test/itkBSplineInterpolationWeightFunctionTest.cxx [hunk 1]
```diff
@@ -145,6 +145,16 @@ itkBSplineInterpolationWeightFunctionTest(int, char *[])
 
     auto function = FunctionType::New();
 
+#if !defined(ITK_LEGACY_REMOVE)
```
Comment (dzenanz) [inline Modules/Core/Common/test/itkBSplineInterpolationWeightFunctionTest.cxx:148]: I guess this sneaked in from the main PR. Either remove or put in a separate commit 😄

Comment (dzenanz) [reply Modules/Core/Common/test/itkBSplineInterpolationWeightFunctionTest.cxx:148]: Oops. Compile error says it needs to be removed.

**Review by dzenanz — APPROVED** [review]

**Review by N-Dekker — APPROVED** [review]
Thank you for moving these small style fixes away from pull request #2860, Hans! 👍 Doing so very much eases reviewing for me 😃
